website/docs/tutorials/08-cairo-1-support/03-cheatcodes/invoke.md

```diff
@@ -0,0 +1,26 @@
+# `invoke`
+
+```cairo
+fn invoke(contract_address: felt252, function_name: felt252, calldata: Array::<felt252>) -> Result::<(), felt252> nopanic;
+```
+
+Invokes a contract. `function_name` parameter should be provided as a shortstring.
```

Comment:
As per @karol-bisztyga suggestion:

```suggestion
Invokes a contract. `function_name` parameter should be provided as a shortstring. It can mutate the state of the contract and does not return a value.  
```

Reply:
And also we could consider mentioning that to query the contract for values you must use `call`?

Reply:
done